resume.sh

```diff
@@ -16,4 +16,12 @@ NAME="${1}"
 
 echo "ssh ${RESOURCE} squeue --name=$NAME --user=$FORWARD_USERNAME -o "%N" -h"
 MACHINE=`ssh ${RESOURCE} squeue --name=$NAME --user=$FORWARD_USERNAME -o "%N" -h`
-ssh -L $PORT:localhost:$PORT ${RESOURCE} ssh -L $PORT:localhost:$PORT -N $MACHINE &
+
+if $SSHBOOL
```

Comment:
The change will need to be made here. `SSHBOOL` does not describe the differences between the two. I wasn't saying to name it that explicitly, I was saying we need a boolean variable that describes the difference to trigger. If someone said "SSHBOOL` is true, nobody would have a clue what that means.

Reply:
Fixed with `$ISOLATEDCOMPUTENODE` boolean.